fairseq_utility.py

```diff
@@ -0,0 +1,78 @@
+import shlex
+import shutil
+import subprocess
+import json, os
```

Comment:
Have we used `json` anywhere?

Reply:
json is required afterwards for the functions defined for evaluation for easy understanding. I will remove that for now.